### DISCUSSION TITLE
Enable USE_RANDOM_DEV_NONCE to generate random devNonce

### DISCUSSION THREAD

**Discussion by jeliasAcklio** [Lora-net/LoRaMac-node#1338] [Q&A]
Hello,
I am using LoRaMac-node library v4.6.0 for my project. As I want to use the 1.0.4 LoRaWAN L2 stack with using random DevNonce for join request, I enable the flag  `USE_RANDOM_DEV_NONCE` in `LoRaMacCrypto.h`, but it seems `SecureElementRandomNumber` function is not defined.

Is that normal ? In v4.4.7 L2 stack, this function is well defined, but no in latest versions.

Thank you for your help, 

Regards

**Comment by dmeehan1968** [comment]
Removed in commit fde7db27471691b5e9c7562f367f6caa18644613. 

Note that v1.0.4 requires that end devices use an incrementing DevNonce, and I suspect the function became redundant (and the conditional code is probably redundant and should have been removed also).  @djaeckle authored the commit.
